### ISSUE TITLE
Clearer visual indication of containment status

### ISSUE THREAD

**Issue by willscott** [subgraph/subgraph-os-issues#33]
It's difficult to determine from the desktop what security / containment is active for which applications. It would be nice to have some visual indication to understand the level of containment to expect for different applications. It also can provide a heads-up if something is started with unexpected lack of containment.  Maybe this could be shown in the task-bar tile, or the window border color.

Perhaps it already is - evince has a 1px black rectangular border around its window and renders with a different border color - if that's the indication of 'rendered through xpra', it would be great to have that shown to the user during installation or of first startup to make it explicit.

**Comment by ioerror** [comment]
Agreed - I assume all the default apps are contained by oz - but what happens when an app is installed and then added to the dock? Is the dock/app launcher for Gnome checking and ensuring oz is properly linked (no, it isn't) in the path? If so, great; if not, the window manager should be able to visually show it. Perhaps by showing that everything that is unsafe looks "Different" and when it is "secure" it looks "normal" by Gnome standards?
